apps/meteor/app/lib/server/functions/saveUser.ts

```diff
@@ -455,8 +452,13 @@ export const saveUser = async function (userId, userData) {
 		performedBy: await safeGetMeteorUser(),
 	});
 
-	if (sendPassword) {
-		await _sendUserEmail(settings.get('Password_Changed_Email_Subject'), passwordChangedHtml, userData);
+	if (sendPassword && userUpdated?.emails && userUpdated.emails[0].address) {
+		userData.email = userUpdated.emails[0].address;
```

Comment:
If this assignment wasn't done before, how this func worked? If we're fixing an issue, we should fix it outside of the ts conversion.

(additionally, i think this must check the first _verified_ email the user has)